firmware/library/L2_Utilities/util.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+
+namespace Utility
+{
+static uint32_t Random(uint32_t min, uint32_t max);
```

Comment:
Remove static from all of the method definitions. Static used in this way limits the function calls to the scope of this file.